src/AutoPilotPlugins/PX4/PowerComponent.qml

```diff
@@ -130,7 +130,7 @@ SetupPage {
                                     QGCTextField { id: measuredVoltage }
 
                                     QGCLabel { text: qsTr("Vehicle voltage:") }
-                                    QGCLabel { text: controller.vehicle.battery.voltage.valueString }
```

Comment:
I don't think this is correct. Current PowerComponent setup ui is only valid for a single battery. How does firmware handle multiple batteries? I would assume each battery has it's own power controller which would mean they could differ?

Reply:
This is how ArduPilot power setup works. There are two separate full sets of parameters for each battery.